src/components/assets/Erc20Currency.vue

```diff
@@ -1,102 +1,209 @@
-              <span class="text--label">{{ token.name }}</span>
-            </div>
+            </svg>
+          </button>
+        </router-link>
+        <q-tooltip>
+          <span class="text--tooltip">{{ $t('assets.send') }}</span>
+        </q-tooltip>
+      </div>
+      <div v-if="token.isWrappedToken && !token.isXC20">
+        <a :href="token.wrapUrl" target="_blank" rel="noopener noreferrer">
+          <button class="btn icon-button">
+            <astar-icon-bridge />
+          </button>
+        </a>
+        <q-tooltip>
+          <span class="text--tooltip">{{ $t('assets.wrap') }}</span>
+        </q-tooltip>
+      </div>
+
+      <div>
+        <a class="box--explorer" :href="explorerLink" target="_blank" rel="noopener noreferrer">
+          <button class="btn icon-button icon-external-link">
+            <astar-icon-external-link />
+          </button>
+        </a>
+        <q-tooltip>
+          <span class="text--tooltip">{{ $t('blockscout') }}</span>
+        </q-tooltip>
+      </div>
+      <div>
+        <button
+          class="btn icon-button icon-plus"
+          @click="
+            addToEvmProvider({
+              tokenAddress: token.address,
+              symbol: token.symbol,
+              decimals: token.decimal,
+              image: token.image,
+              provider,
+            })
+          "
+        >
+          <div>
+            <astar-icon-base>
+              <astar-icon-plus />
+            </astar-icon-base>
           </div>
-        </div>
-        <div class="row__right row__right--evm">
-          <div class="column column--balance">
-            <div class="column__box">
-              <div class="text--accent">
-                <token-balance :balance="token.userBalance" :symbol="token.symbol" />
-              </div>
-              <div class="text--label">
-                <span>{{ $n(Number(token.userBalanceUsd)) }} {{ $t('usd') }}</span>
-              </div>
-            </div>
+          <q-tooltip>
+            <span class="text--tooltip">{{ $t('assets.addToWallet') }}</span>
+          </q-tooltip>
+        </button>
+      </div>
+      <div v-if="isImportedToken" class="">
+        <button
+          class="btn btn--sm btn--delete adjuster--width"
+          @click="handleDeleteStoredToken(token.address)"
+        >
+          <div class="adjuster--width icon--delete">
+            <astar-icon-delete size="22" />
           </div>
-          <div class="column--asset-buttons column--buttons--multi">
-            <div v-if="token.isXC20" />
-            <router-link :to="buildTransferPageLink(token.symbol)">
-              <button class="btn btn--sm">
-                {{ $t('assets.transfer') }}
+        </button>
+        <q-tooltip>
+          <span class="text--tooltip">{{ $t('remove') }}</span>
+        </q-tooltip>
+      </div>
+      <div>
+        <!-- TODO: the icon color will change by class name
```

Comment:
Hi @bobo-k2, I left this kind of comment in my code. For example, this comment is for the "Add to / remove from favourite" feature. Let me know if this works for you.

<img width="1355" alt="Screenshot 2023-09-18 at 5 44 19 PM" src="https://github.com/AstarNetwork/astar-apps/assets/33358068/394e4621-38f7-4d08-ba4d-9a18208a26e7">